pennylane/transforms/defer_measurements.py

```diff
@@ -412,7 +412,7 @@ def _add_control_gate(op, control_wires, reduce_postselected):
                 ctrl(
                     lambda: qml.apply(op.then_op),  # pylint: disable=cell-var-from-loop
                     control=Wires(control),
-                    control_values=branch,
+                    control_values=list(branch),
```

Comment:
With the change to `controlled.py` above, this is a somewhat redundant change that could be skipped or included, either way works.

Reply:
I'd prefer to leave this as it was, but it really comes down to personal preference. Not a blocking change regardless.

Reply:
That's enough for me to revert the change :)